google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractReadContext.java

```diff
@@ -707,6 +713,15 @@ public void close() {
   @Nullable
   abstract TransactionSelector getTransactionSelector();
 
+  /**
+   * Returns the transaction tag for this {@link AbstractReadContext} or <code>null</code> if this
+   * {@link AbstractReadContext} does not have a transaction tag.
+   */
+  @Nullable
+  String getTransactionTag() {
```

Comment:
nit: could we also create a `hasTransactionTag()` that checks if `getTransactionTag() != null`?